coral/src/app/features/approvals/schemas/components/schema-request-table.tsx

```diff
@@ -0,0 +1,101 @@
+import { Box, DataTableColumn, GhostButton, Icon } from "@aivenio/aquarium";
+import deleteIcon from "@aivenio/aquarium/dist/src/icons/delete";
+import tickCircle from "@aivenio/aquarium/dist/src/icons/tickCircle";
+import infoSign from "@aivenio/aquarium/dist/src/icons/infoSign";
+
+interface SchemaRequestTableData {
+  id: number;
+  topicname: string;
+  environmentName: string;
+  username: string;
+  requesttimestring: string;
+}
+
+const columns: Array<DataTableColumn<SchemaRequestTableData>> = [
+  { type: "text", field: "topicname", headerName: "Topic" },
+  { type: "text", field: "environmentName", headerName: "Cluster" },
+  { type: "text", field: "username", headerName: "Requested by" },
+  {
+    type: "text",
+    field: "requesttimestring",
+    headerName: "Date Requested",
+    formatter: (value) => {
+      return `${value} UTC`;
+    },
+  },
+  {
+    type: "custom",
+    // @TODO PR open in DS to be able to
+    // add an "invisible" header name that
+    // is used as aria-label. That will also
+    // solve the duplicate key warning
+    //https://github.com/aiven/design-system/pull/950
+    headerName: "Details",
+    width: 30,
+    UNSAFE_render: (row) => {
+      return (
+        <GhostButton
+          onClick={() => alert("Approve")}
+          title={`View schema request `}
+          dense
+        >
+          <Box
+            component={"span"}
+            display={"flex"}
+            alignItems={"center"}
+            colGap={"2"}
+          >
+            <Icon icon={infoSign} />
+            <span aria-hidden={"true"}>View details</span>
+            <span className={"visually-hidden"}>
```

Comment:
That is nice, I will add it to the ACL table ^^

Reply:
It's not super ideal to be honest, because as far as I remember, there is a screenreader that does ignore `aria-hidden`, which would lead to a redundant additional "View details" read. But I'll to an accessibility check at the end anyway and then check that again, so we can update it if needed!